src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -124,6 +124,9 @@ def _check_conditional_args(
         args = _convert_str_to_enum_entry(args, "backend", OutputFormat)
         args.output_format = args.backend
 
+    if args.service_kind == "tensorrtllm_engine":
+        args.output_format = OutputFormat.TENSORRTLLM_ENGINE
```

Comment:
Can we add one test that checks if the output format is set properly?

Reply:
That's a great idea. I will do that. Thank you

Reply:
Updated existing test to check for the new option and the correct output format